src/components/Dashboard/TodoList.vue

```diff
@@ -701,7 +701,7 @@ export default class TodoList extends Vue {
   /** Loads a NEW Annual Report todo. */
   private loadAnnualReportTodo (task: ApiTaskIF): void {
     const todo = task.task.todo
-    const business = todo.business as BusinessIF
+    const business = todo.business as ApiBusinessIF
```

Comment:
I think the new name makes more sense since it's the business object from the API.